src/agent/command_handler/include/command_handler.hpp

```diff
@@ -132,8 +141,17 @@ namespace command_handler
             {
                 for (auto& cmd : *cmds)
                 {
-                    cmd.ExecutionResult.ErrorCode = module_command::Status::FAILURE;
-                    cmd.ExecutionResult.Message = "Agent stopped during execution";
+                    if (cmd.Command == "restart")
```

Comment:
After rebase, add command name [here](https://github.com/wazuh/wazuh-agent/blob/master/src/agent/command_handler/include/command_entry/command_entry.hpp#L10).

Reply:
Done. Thanks for the feedback and the pointer!